### PR TITLE
[DE-518] DE-535 - Fix PricePoints warnings

### PR DESCRIPTION
**Pull request by patryk-grudzien-keen**

### CODE REVIEW

File: reference/Chargify-API.v1.yaml [hunk 1]
```diff
@@ -17003,9 +17006,7 @@ paths:
           name: "filter[start_datetime]"
           description: "The start date and time (format YYYY-MM-DD HH:MM:SS) with which to filter the date_field. Returns price points with a timestamp at or after exact time provided in query. You can specify timezone in query - otherwise your site's time zone will be used. If provided, this parameter will be used instead of start_date."
         - schema:
-            type: array
-            items:
-              $ref: "../components/schemas/Price-Point-Type.yaml"
+            $ref: "../components/schemas/Price-Point-Type.yaml"
```
Comment (michalpierog) [inline reference/Chargify-API.v1.yaml:17009]: Why this was changed from array? The description and examples suggests it's an array

Comment (patryk-grudzien-keen) [reply reference/Chargify-API.v1.yaml:17010]: Reverted 👍

File: reference/Chargify-API.v1.yaml [hunk 2]
```diff
@@ -16811,23 +16813,25 @@ paths:
...
-                              unit_price: "1.0"
-                              price_point_id: 1
-                              formatted_unit_price: $1.00
-                              segment_id: null
+                      - id: 1
+                        name: "Auto-created"
+                        type: "default"
+                        pricing_scheme: "per_unit"
+                        component_id: 2
+                        handle: "auto-created"
```
Comment (michalpierog) [inline reference/Chargify-API.v1.yaml:16821]: quotes are not required for strings

Comment (patryk-grudzien-keen) [reply reference/Chargify-API.v1.yaml:16821]: There is a mixed approach when it comes to strings so I leave them with quotes.

**Pull request merged**